openmdao/drivers/latinhypercube_driver.py

```diff
@@ -109,10 +109,14 @@ def mmphi(self):
             # TODO: This norm takes up the majority of the computation time. It
```

Comment:
Is this still true, or is this fast enough now that we don't have to bother with compiled options?